canonical-websites/tutorials.ubuntu.com
# Upgrade Polymer and add prpl-server

Update Polymer to 1.11 and enable prpl-server.

This:
 - Updates Polymer and index.html initialisation
 - Installs prpl-server and creates yarn script
 - Improvements to page loading and metadata. Which helps prepare for rendertron.

## QA

 - `./run clean && ./run` - make sure it all functions the same.
- Run locally to test `prpl-server`:
  - `./run clean`
  - `yarn install`
  - `bower install`
  - `yarn run build-all`
  - `yarn run prpl-server`

Have a click around and check the functionality of the site.
